Makefile

```diff
@@ -316,6 +320,18 @@ else
 	PLATFORM_CXXFLAGS += -isystem $(GTEST_DIR)
 endif
 
+ifeq ($(USE_FOLLY_DISTRIBUTED_MUTEX),1)
+       FOLLY_DIR = ./third-party/folly
+       # AIX: pre-defined system headers are surrounded by an extern "C" block
+       ifeq ($(PLATFORM), OS_AIX)
+       PLATFORM_CCFLAGS += -I$(FOLLY_DIR)
+       PLATFORM_CXXFLAGS += -I$(FOLLY_DIR)
+else
+       PLATFORM_CCFLAGS += -isystem $(FOLLY_DIR)
+       PLATFORM_CXXFLAGS += -isystem $(FOLLY_DIR)
+endif
```

Comment:
Hmm.... Looks like indenting is skewed. Likely the patch I gave you was wrong.

Reply:
Yeah, I noticed that, thought it was intentional, I'll fix